examples/command_line/lib/dir with space/lib.h

```diff
@@ -0,0 +1,2 @@
+// function declaration that returns a string
+char *greeting();
```

Comment:
Curious. Was there special handling that had to be added to support spaces in the path?